README.md

```diff
@@ -62,7 +62,7 @@ Configuration
 
 ### Input Scripting
 
-The plugin has a extension called `script` which allows you to do simple scripting automation in a connected device.
+The plugin has a extension called `scripts` which allows you to do simple scripting automation in a connected device.
```

Comment:
I'd suggest:
"The plugin provides an extension called `scripts` which allows to perform simple scripting automations."